omnigibson/controllers/joint_controller.py

```diff
@@ -25,6 +33,9 @@ def __init__(
         dof_idx,
         command_input_limits="default",
         command_output_limits="default",
+        kp=None,
+        damping_ratio=None,
+        use_impedances=True,
```

Comment:
It's a bit unclear how `use_impedances` interacts with `motor_type`. Seems like `use_impedances=not motor_type==EFFORT`?

Reply:
ahhh yeah, sorry, maybe I can update the semantics? Basically use_impedances --> bypass omni position / velocity control, and directly control efforts using our manually set `kp`, `damping_ratio`, and manually include gravity forces, cc forces, etc.